integration

```diff
@@ -679,6 +679,7 @@
   "ReneNulschDE/mbapi2020",
   "rgc99/irrigation_unlimited",
   "rkoebrugge/hacs-youless-component",
+  "rnovacek/homeassistant_cz_energy_spot_prices",
```

Comment:
Missing country https://github.com/rnovacek/homeassistant_cz_energy_spot_prices/blob/main/hacs.json

Reply:
Sorry, I missed that. Added now.